src/types/model.ts

```diff
@@ -117,23 +116,26 @@ export interface Institution {
 
 export interface Dataset {
   name: string;
-  datasetName: string;
+  // datasetName is deprecated, use name instead
```

Comment:
```suggestion
  // @deprecated datasetName is deprecated, use name instead
```

I think you can use the `@deprecated` attribute: https://stackoverflow.com/a/62991642

Reply:
Nice!